diff -ruN reg_tmp/system.reg reg_tmp_fixed/system.reg
--- reg_tmp/system.reg	2018-07-10 09:05:51.000000000 +0800
+++ reg_tmp_fixed/system.reg	2020-09-30 10:49:52.685269410 +0800
@@ -33235,7 +33235,7 @@
 #time=1d375560815cedc
 "DisplayIcon"="C:\\Program Files\\Tencent\\TIM\\TIMUninst.ico"
 "DisplayName"="TIM"
-"DisplayVersion"="2.0.0.22317"
+"DisplayVersion"="3.2.0.21856"
 "EstimatedSize"=dword:0002f7d0
 "HelpLink"="http://www.tencent.com"
 "HelpTelephone"="(86) 755 86060836"
@@ -33259,15 +33259,16 @@
 [Software\\Microsoft\\Windows\\CurrentVersion\\Winlogon] 1488419467
 #time=1d292f775c45ac6
 
-[Software\\Microsoft\\Windows NT\\CurrentVersion] 1508383851
-#time=1d3488aa9ba4da6
-"CSDVersion"="Service Pack 3"
-"CurrentBuildNumber"="2600"
+[Software\\Microsoft\\Windows NT\\CurrentVersion] 1571657811
+#time=1d58803d4d74e20
+"CSDVersion"="Service Pack 1"
+"CurrentBuild"="7601"
+"CurrentBuildNumber"="7601"
 "CurrentType"="Uniprocessor Free"
-"CurrentVersion"="5.1"
+"CurrentVersion"="6.1"
 "InstallDate"=dword:4be5019a
 "ProductId"="12345-oem-0000001-54321"
-"ProductName"="Microsoft Windows XP"
+"ProductName"="Microsoft Windows 7"
 "RegisteredOrganization"=""
 "RegisteredOwner"=""
 "SystemRoot"="C:\\windows"
@@ -34752,7 +34753,7 @@
 
 [Software\\Tencent\\PlatForm_Type_List\\1] 1511754918
 #time=1d367338a03faee
-"TypeName"="TIM2.0.0"
+"TypeName"="TIM3.2.0"
 "TypePath"="c:\\Program Files\\Tencent\\TIM\\Bin\\TIM.exe"
 
 [Software\\Tencent\\QQ2009\\Component] 1488419637
diff -ruN reg_tmp/user.reg reg_tmp_fixed/user.reg
--- reg_tmp/user.reg	2018-07-10 09:05:51.000000000 +0800
+++ reg_tmp_fixed/user.reg	2020-09-30 10:47:13.175262388 +0800
@@ -1024,6 +1024,7 @@
 "qdantidrv.sys"=""
 "qqsetupex.exe"=""
 "riched20"="native,builtin"
+"tencentdl.exe"=""
 "txplatform.exe"=""
 "txupd.exe"=""
 "usp10"="native,builtin"
